nuttx/Kconfig

```diff
@@ -311,10 +311,10 @@ config CHIP_MFG_ID
 		Chip Vendor ID.  For example STM's ID is 00000104
 
 config CHIP_PRODUCT_ID
-	hex "Unique Product ID for chip"
-	default 0x00006435
+	hex "Unique Product ID for chip (STM32L4X3 = 0x435, STM32L4X6 = 0x415)"
+        default 0x415
```

Comment:
Remove this change until we can properly test it and ensure there are no conflicts with the AP.